packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -8871,6 +8893,16 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
     syncScroll: (container: HTMLElement) => void;
 }
 
+// @public (undocumented)
+export enum StickyContainerBehaviorType {
+    Default = 0,
+    StickyAlways = 2,
+    StickyOnScroll = 1
+}
+
+// @public (undocumented)
+export type StickyContainerPosition = 'above' | 'below';

```

Comment:
"Above" and "Below" what? You might want to be clearer here in documentation if nowhere else.